sockeye/arguments.py

```diff
@@ -1067,6 +1067,11 @@ def add_inference_args(params):
                                type=float,
                                help='Beta factor for the length penalty used in beam search: '
                                     '(beta + len(Y))**alpha/(beta + 1)**alpha. Default: %(default)s')
+    decode_params.add_argument('--override-dtype',
+                               default=None,
+                               type=str,
+                               help='EXPERIMENTAL: may be changed or removed in future. Forces dtype of encoders and '
```

Comment:
'Forces' -> 'Enforces' or 'Overrides training dtype of encoders and decoders during inference.'